src/freenet/client/filter/CSSTokenizerFilter.java

```diff
@@ -1620,6 +1621,11 @@ else if("word-wrap".equalsIgnoreCase(element))
 			elementVerifiers.put(element,new CSSPropertyVerifier(Arrays.asList("normal", "break-word", "anywhere"),ElementInfo.VISUALMEDIA));
 			allelementVerifiers.remove(element);
 		}
+		else if("writing-mode".equalsIgnoreCase(element))
+		{
+			elementVerifiers.put(element,new CSSPropertyVerifier(Arrays.asList("horizontal-tb", "vertical-rl", "vertical-lr", "lr", "lr-tb", "rl", "tb", "tb-lr", "tb-rl"),ElementInfo.VISUALMEDIA));
+			allelementVerifiers.remove(element);
```

Comment:
The shorthands are deprecated except for SVG (see https://developer.mozilla.org/en-US/docs/Web/CSS/writing-mode ), so I think we should avoid them.

Supporting SVG as content type would be a huge project, because that can embed far too many ways to access remote content.

Reply:
I think these are valid and not dangerous so should be here. Freenet also has lots of other deprecated things, but still allowed because it's safe.

Reply:
OK.